Docker/pgbouncer/pgbouncer.ini

```diff
@@ -0,0 +1,12 @@
+[databases]
+adaguc = host=adaguc-db port=5432 dbname=adaguc
+
+[pgbouncer]
+listen_port = 5432
+listen_addr = *
+auth_type = trust
+auth_file = /adaguc/pgbouncer/userlist.txt
+logfile = pgbouncer.log
+pidfile = pgbouncer.pid
+server_tls_sslmode = require
```

Comment:
And this gets overwritten locally, correct?

Reply:
Correct, it gets replaced with `disable`.

Running adaguc local with `server_tls_sslmode = require` did not work, and I did not look into configuring SSL for the local/docker postgres instance.